### PR TITLE
New Package: png++-0.2.9

### PR DESCRIPTION
**Pull request by gregf**

### CODE REVIEW

File: srcpkgs/png++/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'png++'
+pkgname=png++
+version=0.2.9
+revision=1
+build_style=gnu-makefile
+makedepends="pkg-config libpng-devel"
+make_install_args="install PREFIX=/usr"
```
Comment (xtraeme) [inline srcpkgs/png++/template]: This line is unnecessary.

File: srcpkgs/png++/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+short_desc="PNG++ aims to provide simple yet powerful C++ interface to libpng"
```
Comment (xtraeme) [inline srcpkgs/png++/template]: Please change this description to `A simple yet powerful C++ interface to libpng`

File: srcpkgs/png++/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+maintainer="Greg Fitzgerald <gregf@voidlinux.eu>"
+license="BSD"
+homepage="http://www.nongnu.org/pngpp/"
+distfiles="http://download.savannah.gnu.org/releases/pngpp/${pkgname}-${version}.tar.gz"
+checksum=abbc6a0565122b6c402d61743451830b4faee6ece454601c5711e1c1b4238791
+
+post_install() {
+				vlicense COPYING
```
Comment (xtraeme) [inline srcpkgs/png++/template]: Fix indentation

File: srcpkgs/png++/template [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'png++'
+pkgname=png++
+version=0.2.9
+revision=1
+build_style=gnu-makefile
+makedepends="pkg-config libpng-devel"
```
Comment (xtraeme) [inline srcpkgs/png++/template]: `pkg-config` must be run in the *host* machine, so that it needs to be moved to `hostmakedepends`

**Comment by xtraeme** [comment]
Also the pkg is empty...

**Comment by dominikh** [comment]
Out of curiosity: Why this package? Will a future package depend on it, or does some proprietary, unpackaged software need it?

**Comment by gregf** [comment]
I was going to use this for a future package, but I have sense decided against using it. Sorry to waste your time @xtraeme.

**Pull request closed without merging**